flake8_sorted_keys/checker.py

```diff
@@ -1,13 +1,35 @@
 __version__ = '0.0.1'
 
+import ast
+
 
 class SortedKeysChecker(object):
     name = 'flake8-sorted-keys'
     version = __version__
 
+    unsorted_message = 'S001 Sort keys'
+
     def __init__(self, tree, *args, **kwargs):
-        pass
+        self.tree = tree
+
+    def needs_checking(self, dict_node):
+        """Decide weather specific Dict literal node needs to be considered.
+
+        We are only interested in multi-line dicts with all string keys.
+        """
+        if not all(isinstance(key, ast.Str) for key in dict_node.keys):
+            return False
```

Comment:
We should think over later what to do with the case when not all keys are strings.